crate_universe/tests/integration.rs

```diff
@@ -13,6 +13,7 @@ use crate_universe_resolver::config::{Config, Override, Package};
 use crate_universe_resolver::NamedTempFile;
 
 #[test]
+#[ignore]  // TODO: This test fails because of the way `test` is implemented, see that method for details
```

Comment:
Let's delete this whole file, no point keeping around a whole file of broken tests :)